Cargo.toml

```diff
@@ -15,6 +15,10 @@ documentation = "https://docs.rs/cupid/"
 license = "MIT"
 build = "build.rs"
 
+[features]
+default = ["std"]
+std = []
+
```

Comment:
If `lib.rs` doesn't do something like `#![cfg_attr(feature = "std", no_std)]`, there's really no point in having this. If it's using `core` by default anyway, I think it's fine to just delete this instead of adding a bunch of conditional compilation.

Reply:
I agree, and I thought I deleted this. Please feel free to delete it, squash with the current version, and force-push.